.github/pull_request_template.md

```diff
@@ -0,0 +1,5 @@
+### Motivation
```

Comment:
Does this interfere with Github taking the long description part of a commit message and using that as the PR description?

Reply:
I'm not sure, I suspect it does. From a quick look at PR's we've merged the past couple weeks it appears we don't use that feature very often.

Reply:
I'm probably the only person who writes descriptions of changes in the commit messages. As such I'd prefer to keep the current behaviour so the commit message becomes the PR text.

Reply:
Alright, in that case would dropping the PR template and keeping the issue template be a reasonable compromise?

Reply:
Sure